lib/roo/excelx.rb

```diff
@@ -23,7 +23,7 @@ class Excelx < Roo::Base
     require 'roo/excelx/coordinate'
     require 'roo/excelx/format'
 
-    delegate [:styles, :workbook, :shared_strings, :rels_files, :sheet_files, :comments_files] => :@shared
```

Comment:
I'd actually prefer to used delegate for these methods and delete lines 429-441.